config/disabled.yml

```diff
@@ -5,6 +5,10 @@
 Rails:
   Enabled: false
 
+Rails/OutputSafety:
+  Description: 'The use of `html_safe` or `raw` maybe a security risk.'
+  Enabled: false
```

Comment:
It seemed pretty opinionated by default. There are going to be some places you'll need to use this. For our app, I'm just taking the stance that all those places should use an explicit `# rubocop:disable` comment to whitelist those few spots.